### ISSUE TITLE
HTTP 400 After Upgrade to Splunk 6.0

### ISSUE THREAD

**Issue by broamski** [splunk/splunk-sdk-java#53]
Hello,

I'm experiencing the following error when attempting to connect to my upgraded Splunk 6.0 instance via the Java client:

[Fatal Error] :1:3: The markup in the document preceding the root element must be well-formed.
Exception in thread "main" com.splunk.HttpException: HTTP 400
    at com.splunk.HttpException.create(HttpException.java:59)
    at com.splunk.HttpService.send(HttpService.java:355)
    at com.splunk.Service.send(Service.java:1203)
    at com.splunk.HttpService.post(HttpService.java:212)
    at com.splunk.Service.login(Service.java:1040)
    at com.splunk.Service.login(Service.java:1020)
    at com.splunk.Service.connect(Service.java:161)
    at Thing.main(Thing.java:28)

Code Snippet:

ServiceArgs loginArgs = new ServiceArgs();
loginArgs.setUsername(username);
loginArgs.setPassword(password);
loginArgs.setHost(hostname);
loginArgs.setPort(serviceport);

Service service = Service.connect(loginArgs);

String outputMode = "xml";// xml,json,csv
//String searchQuery  = "search index=sweet sourcetype=dude";

Args queryArgs = new Args();
queryArgs.put("earliest_time", "-1m@m");
queryArgs.put("latest_time", "@m");
queryArgs.put("output_mode", outputMode);
queryArgs.put("count", 0);

InputStream stream = service.oneshotSearch(searchQuery,
        queryArgs);

**Comment by shakeelmohamed** [comment]
Hi @smilingleo 

Sorry we didn't see this before, if you're still blocked on this please open a new issue.
If you found a solution, please let us know

**Comment by Marcwe25** [comment]
how can i solved this ?
`[Fatal Error] :1:3: The markup in the document preceding the root element must be well-formed.
Exception in thread "main" java.lang.RuntimeException: The markup in the document preceding the root element must be well-formed.
	at com.splunk.Xml.parse(Xml.java:63)
	at com.splunk.Xml.parse(Xml.java:40)
	at com.splunk.Service.login(Service.java:1125)
	at com.splunk.Service.login(Service.java:1103)
	at com.splunk.Service.connect(Service.java:189)
	at splsdk.Main.main(Main.java:21)
Caused by: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 3; The markup in the document preceding the root element must be well-formed.
	at com.sun.org.apache.xerces.internal.parsers.DOMParser.parse(DOMParser.java:257)
	at com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderImpl.parse(DocumentBuilderImpl.java:339)
	at javax.xml.parsers.DocumentBuilder.parse(DocumentBuilder.java:121)
	at com.splunk.Xml.parse(Xml.java:60)`

the code is:

`public static void main(String[] args) {
 	HttpService.setSslSecurityProtocol(SSLSecurityProtocol.TLSv1_2);
// Create a map of arguments and add login parameters
ServiceArgs loginArgs = new ServiceArgs();
loginArgs.setUsername("******");
loginArgs.setPassword("*******");
loginArgs.setHost("*****");
loginArgs.setPort(443);
// Create a Service instance and log in with the argument map
Service service = Service.connect(loginArgs);`

**Comment by shakeelmohamed** [comment]
@Marcwe25 can you send us an email devinfo@splunk.com with some additional information: versions of Splunk, Java, the SDK and if you're using Splunk Cloud.
